src/core/server/saved_objects/service/lib/repository.test.js

```diff
@@ -2115,7 +2130,7 @@ describe('SavedObjectsRepository', () => {
       ).rejects.toEqual(new Error('Unsupported saved object type: \'hiddenType\': Bad Request'));
     });
 
-    it('should not return hidden saved ojects when attempting to \'find\' support and unsupported types', async () => {
+    it('should return an error object when attempting to \'find\' support and unsupported types', async () => {
```

Comment:
@mikecote In all the other SavedObject functions we throw an error when you try to use an unsupported type, do you know if there was a reason `find` returns an empty response instead of throwing?

There's another inconsistency in that delete() doesn't return an UnsupportedTypeError but a generic 404. I'm just scared there was a reason to deviate from the other functions.

Reply:
For security, we put it back the way it was. We might create another PR later on to throw an error.